test/integration/http.spec.js

```diff
@@ -1,150 +0,0 @@
-/* dependencies */
```

Comment:
Remove this comment

Reply:
Renamed this file to jurisdiction.http.spec.js